packages/docs/guide/index.md

```diff
@@ -73,6 +73,7 @@ app.mount('#app')
 
 // Now the app has started!
 ```
+**Note: Calling `app.use(router)` will triggers the initial navigation**
```

Comment:
Let's add it to the sentence below instead: `By caling `app.use(router)`, we are triggering the initial navigation and giving access to `this.$router` [...](rest of the sentence)